v2/dtos/corecommand.go

```diff
@@ -0,0 +1,17 @@
+//
+// Copyright (C) 2021 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package dtos
+
+// CoreCommand and its properties are defined in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-command/2.x#/CoreCommand
+type CoreCommand struct {
+	Name       string `json:"name" validate:"required,edgex-dto-none-empty-string,edgex-dto-rfc3986-unreserved-chars"`
+	DeviceName string `json:"deviceName" validate:"required,edgex-dto-rfc3986-unreserved-chars"`
+	Get        bool   `json:"get,omitempty" validate:"required_without=Put"`
+	Put        bool   `json:"put,omitempty" validate:"required_without=Get"`
```

Comment:
Get and Put should be both required, and they can not be both false.
This validation rule also applies to Command DTO

Reply:
Okay, after looking into https://pkg.go.dev/gopkg.in/go-playground/validator.v9, I think we need a custom tag to achieve this validation.

Reply:
We don't have to create a custom tag for this, since it is an easy check and might not be reused.
Please just add the check in its Validate() func like this:
https://github.com/edgexfoundry/go-mod-core-contracts/blob/1915b16eb2eb9b1cb835de2bb0cf2b3eddccddfb/v2/dtos/requests/deviceprofile.go#L28-L34

Reply:
Per my experimentation and discussion, we actually don't have to add extra codes to validate false GET and false PUT case, as `required_without` tag could actually satisfy our validation need.

Note:  the [go doc for required_without](https://pkg.go.dev/gopkg.in/go-playground/validator.v9#hdr-Required_Without) is misleading, and should be revised to: 
```
The field under validation must be not the data types default zero value and not empty only when any of the other specified fields are not the data types default zero value and not empty.
```

GET and PUT are defined as bool, which has false as default value, so that following struct could actually throw validation error from go validator when Get=false and Put=false.
```
Get        bool   `json:"get" validate:"required_without=Put"`
Put        bool   `json:"put" validate:"required_without=Get"`
```